app/src/test/contrib/components/Responsibilities/BurdenEstimates/PopulateBurdenEstimateFormComponentTests.tsx

```diff
@@ -213,15 +213,6 @@ describe("Populate Burden Estimates Form Component tests", () => {
         expect(newState.file.fileName).to.eq("test.csv")
     });
 
-    it("shows success message on file population success", () => {
```

Comment:
This component is actually immediately hidden on success, so the success message should not be shown here.